client/src/Components/Controls.js

```diff
@@ -0,0 +1,59 @@
+import React, { Component } from 'react';
+import io from 'socket.io-client';
+import Url from 'url-parse';
+
+class Controls extends Component {
```

Comment:
Saw you created components, so I moved everything I did into a controls component but I dont know how to get this to be displayed....

Reply:
figured this out it was actually pretty similar to how angular works.